rust-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+# Licensed under the Apache-2.0 license
+
+[toolchain]
+channel = "1.70"
+targets = ["riscv32imc-unknown-none-elf"]
```

Comment:
This is neat. I wasn't aware of `rust-toolchain.toml`.

Do we still need the "Install risv32imc target" build step in CI?

Reply:
Good idea. Removed.